### PR TITLE
Update README.md

### LINKED ISSUE #4053: Readme Qt installation doesn't list Debian dependencies
I had no errors building, but couldn't start the GUI until I installed `qml-module-qt-labs-platform`:
```
W QQmlApplicationEngine failed to load component
W qrc:/main.qml:2378:5: Type MoneroComponents.MenuBar unavailable
W qrc:/components/MenuBar.qml:29:1: module "Qt.labs.platform" is not installed
E Error: no root objects
```

Debian 11.5 does not offer the `qt5-default` package, but all others listed under the Ubuntu dependencies in the "Install Qt" section are available, and on my machine were already installed (with the exception of `qml-module-qt-labs-platform`).

`qt5-default` is an Ubuntu-specific metapackage: https://packages.ubuntu.com/focal/qt5-default
Thus, I would suggest replacing it with `qtbase5-dev qtchooser` (both of which are available on Debian) and changing the section header to "For Debian distributions (Debian, Ubuntu, Mint, Tails...)" as in the "Install Monero dependencies" section.

### PR DESCRIPTION
**Pull request by 2l47**
Resolves #4053

### CODE REVIEW

**Comment by 2l47** [comment]
Would appreciate it if someone running Mint/Tails could verify the presence of all listed packages on their distro.

File: README.md [hunk 1]
```diff
@@ -221,15 +221,15 @@ Packaging for your favorite distribution would be a welcome contribution!
...
 
 The following instructions will fetch Qt from your distribution's repositories instead. Take note of what version it installs. Your mileage may vary.
 
-  - For Ubuntu 17.10+
+  - For Debian distributions (Debian, Ubuntu, Mint, Tails...)
 
-    `sudo apt install qtbase5-dev qt5-default qtdeclarative5-dev qml-module-qtqml-models2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-dialogs qml-module-qtquick-xmllistmodel qml-module-qt-labs-settings qml-module-qt-labs-platform qml-module-qt-labs-folderlistmodel qttools5-dev-tools qml-module-qtquick-templates2 libqt5svg5-dev`
+    `sudo apt install qtbase5-dev qtbase5-dev qtchooser qtdeclarative5-dev qml-module-qtqml-models2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-dialogs qml-module-qtquick-xmllistmodel qml-module-qt-labs-settings qml-module-qt-labs-platform qml-module-qt-labs-folderlistmodel qttools5-dev-tools qml-module-qtquick-templates2 libqt5svg5-dev`
```
Comment (selsta) [inline README.md:232]:
```suggestion
    `sudo apt install qtbase5-dev qtchooser qtdeclarative5-dev qml-module-qtqml-models2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-dialogs qml-module-qtquick-xmllistmodel qml-module-qt-labs-settings qml-module-qt-labs-platform qml-module-qt-labs-folderlistmodel qttools5-dev-tools qml-module-qtquick-templates2 libqt5svg5-dev`
```

**Comment by 2l47** [comment]
Thanks for spotting that.

**Comment by selsta** [comment]
I'm confused how `qtchooser` installs the missing `qml-module-qt-labs-platform`.

**Comment by 2l47** [comment]
`qml-module-qt-labs-platform` was already listed in the dependencies. I assume it's a new dependency that I hadn't installed since I last built monero-gui. My mentioning it was off-topic; I just wanted to point it out for anyone running into the same error, as the build was successful without it, but the GUI wouldn't render.

The dependency list previously included `qt5-default`, which is an Ubuntu-specific metapackage containing `qtbase5-dev` and `qtchooser`. `qtbase5-dev` was already listed in the dependencies. The `qt5-default` metapackage is not available on other Debian-based distributions, so we replace it with only `qtchooser`. I hope that clears things up.

**Comment by selsta** [comment]
I see. Now the only question remaining is if `qtchooser` is required? Based on the description it should not be.

**Comment by selsta** [comment]
Can you update the PR to remove `qtchooser` and rename the commit to something like `README: update Debian dependencies` ? Then I can approve it.

**Comment by 2l47** [comment]
Done.

**Review by selsta — APPROVED** [review]

**Pull request merged**